uber/configspec.ini

```diff
@@ -377,6 +361,10 @@ __many__ = integer
 __many__ = integer
 
 
+[table_prices]
```

Comment:
I believe so, yes.  To be honest, I haven't tested that code since we aren't using it for MAGStock, but obviously we'll be testing it out by the time MagClassic prereg opens.  (Or when we launch MagPrime prereg, whichever comes first.)